README.md

```diff
@@ -19,28 +19,32 @@ This crate is `#[no_std]` compatible with `default-features = false`
 | Feature                | Default? | Description |
 | :---                   | :---     | :---        |
 | `alloc`                | ✓        | Enables features that require dynamic heap allocation |
-| `std`                  | ✓        | std::error::Error types |
-| `zeroize`              | ✓        | Enables `Zeroize` for `SigningKey` |
-| `asm`                  |          | Assembly implementation of SHA-2 compression functions |
-| `batch`                |          | Batch verification. Requires `alloc` |
-| `digest`               |          | TODO |
-| `legacy_compatibility` |          | See: A Note on Signature Malleability |
-| `pkcs8`                |          | PKCS#8 Support |
-| `pem`                  |          | PEM Support |
-| `rand_core`            |          | TODO        |
+| `std`                  | ✓        | Implements `std::error::Error` for `SignatureError` |
+| `zeroize`              | ✓        | Implements `Zeroize` and `ZeroizeOnDrop` for `SigningKey` |
+| `rand_core`            |          | Enables `SigningKey::generate` |
+| `batch`                |          | Enables `verify_batch` for verifying many signatures quickly. Also enables `rand_core`. |
+| `digest`               |          | Enables `Context`, `SigningKey::{with_context, sign_prehashed}` and `VerifyingKey::{with_context, verify_prehashed, verify_prehashed_strict}` for Ed25519ph prehashed signatures |
+| `asm`                  |          | Enables assembly optimizations in the SHA-512 compression functions |
+| `pkcs8`                |          | Enables [PKCS#8](https://en.wikipedia.org/wiki/PKCS_8) serialization/deserialization for `SigningKey` and `VerifyingKey` |
+| `pem`                  |          | TODO |
```

Comment:
```suggestion
| `pem`                  |          | Enables PEM serialization support for PKCS#8 private keys and SPKI public keys |
```

As to why they're separate features: `pkcs8` works in heapless environments. `pem` (currently) requires `alloc`.

Streaming encoding/decoding is almost working on the latest `der`/`pkcs8` prereleases though, which would eliminate that restriction so they don't need to be separate features.

With weak feature activation, it would also be possible to make `pkcs8` + `alloc` automatically enable `pem`.